keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueInputFormat.java

```diff
@@ -67,8 +67,9 @@
     List<InputSplit> splits = Lists.newArrayList();
     try {
       for (Path path : paths) {
-        List<PartitionInfo> partitionInfos =
-            mKeyValueMasterClient.getPartitionInfo(new TachyonURI(path.toString()));
+        TachyonURI tachyonURI = new TachyonURI(path.toString());
```

Comment:
is the purpose of this change to use TachyonURI to drop scheme and authority information? Why do we do this?